modules/programs/eww.nix

```
@@ -23,12 +21,74 @@ in {
       '';
     };
 
-    configDir = mkOption {
-      type = types.path;
-      example = literalExpression "./eww-config-dir";
+    yuck = {
+      enable = mkEnableOption ""
+      mkOption {
+        type = types.nullOr types.lines;
+        default = null;
```

```suggestion
        type = types.lines;
        default = "";
```

Same applies for the `scss` option.

---

I have replied but I think I will keep that because of this comment: https://github.com/nix-community/home-manager/pull/5979?notification_referrer_id=NT_kwDOBUkEqrQxMjk1NTI0MzIyMDo4ODY3MTQwMg#discussion_r1826307253

---

Null is better to avoid overriding